modules/userId/eids.js

```diff
@@ -50,7 +50,7 @@ const USER_IDS_CONFIG = {
 
   // ftrack
   'ftrackId': {
-    source: 'flashtalking.com',
+    source: 'flashtalking.com/identity-framework#FTrack',
```

Comment:
the source should just be a domain. `flashtalking.com` is all you need here

Reply:
Edited